src/main/java/aquality/selenium/core/logging/Logger.java

```diff
@@ -30,7 +36,9 @@ public static Logger getInstance() {
      * @return logger instance
      */
     public Logger addAppender(Appender appender) {
-        log4J.get().addAppender(appender);
+        appender.start();
+        LoggerContext.getContext(false).getRootLogger().addAppender(appender);
+        LoggerContext.getContext().updateLoggers();
```

Comment:
isn't context the same? why are we passing "false" in the first line and nothing in the second line?

Reply:
fx: Removed updateLoggers function (it's getting updated automatically in right context that's why it worked with the wrong context before)